tron/core/recovery.py

```diff
@@ -0,0 +1,63 @@
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+import logging
+
+from tron.actioncommand import NoActionRunnerFactory
+from tron.core.actionrun import ActionRun
+from tron.core.actionrun import SSHActionRun
+
+RECOVERY_BINARY = "/work/bin/recover_batch.py"
```

Comment:
call this tron_recover_action or something, the normal python scripts should help make this work with the links file trick to give you a consistent full path.

Or you could do something like using
    remote_exec_path: "/opt/venvs/tron/bin"
to get the path